### ISSUE TITLE
Driver error

### ISSUE THREAD

**Issue by Abbanan** [EnergyCube/CoWFC#12]
Hi! when I try to go to the website at the end of the install I always get same messgae
"There was an error: could not find driver"
Do you think you can helping fix?

**Comment by EnergyCube** [comment]
Probably related to the PHP version problem.
The repo currently use an outdated version of PHP that isn't present on some distro, making the install process not work properly.

**Comment by Abbanan** [comment]
I see... If I using debian how do suggest going about to fixing this? I think you say for ubuntu replacing mod3="php7.1" with mod3="php7.4" however for debian it seems to be use php7.4 already

**Comment by EnergyCube** [comment]
Yeah, debian should work since it use a more recent version of PHP.
Technically you could also change the version in the Ubuntu script but I didn't got the time to check if it work.

**Comment by itschiel** [comment]
So, this problem also occurs on Debian 11 with PHP version 7.4. I just updated to PHP version 8.1, but that didn't make any difference. Is there a fix for this problem?
